### ISSUE TITLE
Hardcoded six

### ISSUE THREAD

**Issue by urishalit** [benoitc/gunicorn#2080]
Hi There,
There is a known [issue](https://github.com/spotify/luigi/issues/1706) when using pinject and six < 1.9.0. When running with gunicorn there is the problem that six.py is under gunicorn package and not as a dependency. I see in the latest version 19.9.0 it still is the case, but in master it isn't the case anymore.

Is a version expected to remove this dependency? Why is it there from the first place and not list it as a dependency?

Basically, I can't use gunicorn & pinject together currently.

**Comment by urishalit** [comment]
I forked the repository and removed the hardcoded dependency of `six` and it seems to work, opened a PR to merge it back.

https://github.com/benoitc/gunicorn/pull/2083

**Comment by benoitc** [comment]
#2083 has been closed, closing the issue since no activity happened since. Feel free to reopen it if needed. Thanks for the PR also :)
